internal/app/machined/internal/sequencer/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -135,6 +154,10 @@ func (d *Sequencer) Boot() error {
 			configtask.NewExtraFilesTask(),
 			configtask.NewSysctlsTask(),
 		),
+		phase.NewPhase(
```

Comment:
Leaving this mounted as RO at the end here because other sequences like Upgrade want to read this file immediately when they start running.